guides/request-validation-sinatra/example-1/example-1.3.x.rb

```diff
@@ -0,0 +1,17 @@
+require 'sinatra'
+require 'twilio-ruby'
+require 'rack'
+require 'dotenv'
```

Comment:
I don't think `dotenv` is required in the scope of this guide. When you introduce something `dotenv` you should also introduce a **.env** file which holds the env variables. For the purpose of this guide that would be unnecessary complexity.

What do you think?